### ISSUE TITLE
Flutter 3.10.0 seems to reintroduce the bug solved by the hotfix applied in 3.7.11

### ISSUE THREAD

**Issue by adplant** [flutter/flutter#126724]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

See ticket https://github.com/flutter/flutter/issues/124433

### Expected results

Correct build of iOS app

### Actual results

```
Failed to build iOS app
Error (Xcode): File not found: /Applications/Xcode-14.3.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/arc/libarclite_iphoneos.a
```

### Code sample

<details><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
[Paste your output here]
```

</details>

**Comment by carasen12** [comment]
Thank you for providing the necessary information and following the guide
to filing a bug. It seems that the issue you are experiencing has already
been reported in ticket https://github.com/flutter/flutter/issues/124433.
Please continue to follow that thread for updates on the issue. If you have
any further questions or concerns, please let us know.
<https://gpt.space/gmail?source=watermark>
📱

**Comment by adplant** [comment]
@carasen12 I linked directly to that issue, basically stating that the problem has resurfaced in Flutter 3.10.0. That issue has been closed, but either needs to be re-opened or a new one created.

**Comment by zanderso** [comment]
https://github.com/flutter/flutter/commit/c60f202308cbda8f97c0c0f66521c147777101d5 landed after the 3.10 branch cut, and we failed to CP it into both 3.7 stable, and 3.10 while it was still in beta. https://github.com/flutter/flutter/issues/124529. cc @stuartmorgan FYI @CaseyHillers.

**Comment by stuartmorgan** [comment]
This is a duplicate of https://github.com/flutter/flutter/issues/126449
